### PR TITLE
Add: controller to delete client from database

### PR DESCRIPTION
**Pull request by biohazard2117**
Added the controller to delete client from database using clientID

### CODE REVIEW

**Review by VighneshManjrekar — CHANGES_REQUESTED** [review]
1. Use await and try...catch block 
2. Include error handling response
3. Include status in response

**Comment by biohazard2117** [comment]
Changed the code as requested. @VighneshManjrekar

File: controllers/main.controllers.js [hunk 1]
```diff
@@ -41,6 +41,18 @@ exports.deleteProject = async (req, res, next) => {
 // @route   DELETE api/client/:id
 exports.deleteClient = async (req, res, next) => {
   // Delete client with req.params.id
+  try {
+    const { id: _id } = req.params;
+    const docs = await Client.findByIdAndDelete(_id);
+    // console.log(docs);
+    if(docs != null) res.status(200).json(docs);
+    else {
+      res.status(200).json("No client found");
```
Comment (VighneshManjrekar) [inline controllers/main.controllers.js]: Not found should have 404 status

File: controllers/main.controllers.js [hunk 2]
```diff
@@ -41,6 +41,18 @@ exports.deleteProject = async (req, res, next) => {
...
+    }
+  } catch (err) {
+    // console.log(err);
+    res.status(400).json(err);
+  }
 };
```
Comment (VighneshManjrekar) [inline controllers/main.controllers.js:54]: remove log comments rest of code is fine
also format response like this
`{
success: true,
data:{
//data goes here}
}`

File: controllers/main.controllers.js [hunk 3]
```diff
@@ -41,6 +41,18 @@ exports.deleteProject = async (req, res, next) => {
...
+  try {
+    const { id: _id } = req.params;
+    const docs = await Client.findByIdAndDelete(_id);
+    // console.log(docs);
+    if(docs != null) res.status(200).json(docs);
+    else {
+      res.status(200).json("No client found");
+    }
+  } catch (err) {
+    // console.log(err);
```
Comment (VighneshManjrekar) [inline controllers/main.controllers.js]: for error response should be 
`{
success: false, error: {//err message goes here}
}`
send err.message string

**Review by VighneshManjrekar — CHANGES_REQUESTED** [review]

File: controllers/main.controllers.js (hunk 3 above)
Comment (biohazard2117) [reply controllers/main.controllers.js]: @VighneshManjrekar the requested changes have been made. Please review and verify if further changes are needed.

**Pull request merged**